src/madhuraMJun20/scannerClass/PalindromeNumber.java

```diff
@@ -0,0 +1,38 @@
+package madhuraMJun20.scannerClass;
+
+/*Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.
+
+Input: number = 56789
+Output : Given number 56789 is not palindrome.
+*/
+public class PalindromeNumber {
+	int number;
+	int reverseNum;
+
+	int findPalindromeNum(int number) {
+		int mod = 0;
+		while (number > 0) {
+			mod = number % 10;
+			number = number / 10;
+			reverseNum = mod + reverseNum * 10;
+		}
+		return reverseNum;
```

Comment:
No need to return reverseNum value. if you have already declared the reverseNum as instance variable. I suggest to make these changes.

Reply:
Ok thanks. Updated the code